openupgrade_scripts/scripts/hr_holidays/16.0.1.5/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Coop IT Easy (https://coopiteasy.be)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def force_recompute_allocation_validation_type(env):
+    hr_leave_type_ids = (
+        env["hr.leave.type"].with_context({"active_test": False}).search([])
+    )
+    hr_leave_type_ids._compute_allocation_validation_type()
```

Comment:
This is not performant and can lead to unexpected changes, triggering other computes including this one in depends, so it's advisable to do the equivalent one in SQL.

Reply:
Thank you for the advice. It's now done by SQL directly. :)